feat: add project block for basic functions The Odin Project (#54375)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/16-the-odin-project/top-basic-function-projects/top-basic-functions-exercise-c.md

---
id: 661e151f068359c3ccf2f4d7
title: Basic Functions Exercise C
challengeType: 1
dashedName: top-basic-functions-exercise-c
---

# --description--

> Write a function, named `capatalize`, that takes a string as an argument and returns the string with the first letter capitalized.

Reviewer: spelling error

```suggestion
Write a function, named `capitalize`, that takes a string as an parameter and returns a new string with the first letter capitalized.
```

# --hints--

> You should have a function named `capatalize`.

Reviewer: ```suggestion
You should have a function named `capitalize `.
```

```js

> assert.strictEqual(capatalize('hello') , 'Hello');

Reviewer: here we should just check that they created the function instead of if they completed it or not.

```suggestion
assert.isFunction(capitalize);
```

```

> Your function should take in a string as an argument. 

Reviewer: ```suggestion
Your function should take in a string as a parameter. 
```

```js

> assert.strictEqual(capatalize('world'), 'World');

Reviewer: ```suggestion
assert.strictEqual(capitalize('world'), 'World');
```

```

> You should return the string with the first letter capitalized.

Reviewer: ```suggestion
Your function should return a new string with the first letter capitalized.
```

```js

> assert.strictEqual(capatalize('sem'), 'Sem');

Reviewer: ```suggestion
assert.strictEqual(capitalize('sem'), 'Sem');
```

```

# --seed--

## --seed-contents--

```js

```

## --solutions--

```js

> function capatalize(str) {

Reviewer: ```suggestion
function capitalize(str) {
```
